contrib/builder/deb/amd64/generate.sh

```diff
@@ -42,11 +42,19 @@ for version in "${versions[@]}"; do
 	echo >> "$version/Dockerfile"
 
 	if [ "$distro" = "debian" ]; then
-		cat >> "$version/Dockerfile" <<-'EOF'
-			# allow replacing httpredir mirror
-			ARG APT_MIRROR=httpredir.debian.org
-			RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
-		EOF
+		if [ "$suite" = "stretch" ]; then
+			cat >> "$version/Dockerfile" <<-'EOF'
+				# allow replacing deb mirror
+				ARG APT_MIRROR=deb.debian.org
+				RUN sed -i s/deb.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
For reference, timeline of apt mirror hostname:

* 2013-11-30 `ftp.us.debian.org`
* 2016-01-14 `httpredir.debian.org`
* 2016-08-29 `deb.debian.org`

According to git commit history of what looks like documentation example of default sources to use for apt mirror hostname:
https://github.com/Debian/apt/commits/master/vendor/debian/sources.list.in